app/javascript/components/pages/TripShow.js

```diff
@@ -17,9 +17,17 @@ class TripShow extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      success: false
+      success: false,
```

Comment:
I don't see success ever being updated or referenced on this page, what are you using success for?